### ISSUE TITLE
PGP signature on Maven Central seems to be incompatible with current Gradle/bouncycastle versions

### ISSUE THREAD

**Issue by cseewald** [kagkarlsson/db-scheduler#518]
## Expected Behavior

I would like to be able to verify db-scheduler.jar with Gradle Dependency Verification with the PGP signature on Maven Central.

## Current Behavior

If I enable dependency verification (https://docs.gradle.org/8.5/userguide/dependency_verification.html) I get an exception in the gradle build which hints at a broken/incompatible PGP signature. 

`class org.bouncycastle.openpgp.PGPCompressedData cannot be cast to class org.bouncycastle.openpgp.PGPSignatureList (org.bouncycastle.openpgp.PGPCompressedData and org.bouncycastle.openpgp.PGPSignatureList are in unnamed module of loader org.gradle.initialization.MixInLegacyTypesClassLoader @15a04efb)`

The error is only reported if db-scheduler is added as a dependency and only if it is version >= 12.0.1.

See https://github.com/cseewald/db-scheduler-client

## Context
- DB-Scheduler Version : 14.0.3
- Java Version : 17
- Spring Boot (check for Yes) : [ ]
- Database and Version : none

**Comment by kagkarlsson** [comment]
Would you please check if the issue persists in `v16.8.1`?

**Comment by cseewald** [comment]
Yes works thx (Java 21, Dependency Verification with Gradle 8.14).

If you want to do this admittedly niche trust thing properly (prevent supply chain attacks) you would need to publish your public key e.g. as a Gist on GitHub (like https://gist.github.com/olamy/3da76b2ca3e91c03acd8). 

Then I know as a user of your lib that the public key I trust in my project comes from you and the artifact on maven central (or from an intermediary) was actually signed by this key.

**Comment by kagkarlsson** [comment]
Great 👍 
I am all for supply chain verification. That looks a bit ad-hoc, how would you go about verifying these? (it doesn't look automatable?)

**Comment by kagkarlsson** [comment]
I am closing this as fixed, but I am curious about what can be done to enable supply chain verification, but need separate issue.
